app/services/automation_observers/all_checked_steps_automation_observer.rb

```
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module AutomationObservers
+  class AllCheckedStepsAutomationObserver
+    def initialize(my_module, user)
+      @my_module = my_module
+      @user = user
+    end
+
+    def call
+      return if @my_module.blank?
```

it can hide potential errors